spec/bolt/node/ssh_spec.rb

```diff
@@ -139,43 +138,14 @@ def result_value(stdout = nil, stderr = nil, exit_code = 0)
     end
   end
 
-  context "when executing with private key" do
```

Comment:
Why remove these tests? They seem useful.

Reply:
Fixed, though now the host key error test is failing and I cannot figure out why....